app/code/community/Algolia/Algoliasearch/etc/system.xml

```diff
@@ -288,7 +304,8 @@
                             <show_in_store>1</show_in_store>
                             <comment>
                                 <![CDATA[
-                                Configure here the pages you don't want to search in.
+                                Configure here the pages you don't want to search in. <br>
+                                Pages' indexing documentation: <a target="_blank" href="https://community.algolia.com/magento/doc/m1/indexing/?utm_source=magento&utm_medium=extension&utm_campaign=magento_1&utm_term=shop-owner&utm_content=doc-link#pages-indexing">https://community.algolia.com/magento/doc/m1/indexing/#pages-indexing</a>
```

Comment:
I would go for singular _"page indexing"_  but I think plural form is correct too 👍